packages/json_glib.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Json_glib < Package
+  description 'JSON-GLib implements a full suite of JSON-related tools using GLib and GObject.'
+  homepage 'https://wiki.gnome.org/Projects/JsonGlib'
+  version '1.4.2'
+  source_url 'https://download.gnome.org/sources/json-glib/1.4/json-glib-1.4.2.tar.xz'
+  source_sha256 '2d7709a44749c7318599a6829322e081915bdc73f5be5045882ed120bb686dc8'
+
```

Comment:
Do we need jsonc?  I guess a fresh crew install would determine it.

Reply:
Hmm, good point. Want me to check?

Reply:
Might as well if it's not too much trouble.

Reply:
Yep, I'll check now!

Reply:
... or in like 5 minutes when crew finishes reinstalling.
we really should provide "core" as a tarball or something, to speed things up

Reply:
Sounds like a good idea.  It would be great to "cache" it also for subsequent installs.

Reply:
It just built successfully without jsonc, so we're good.